lib/publiccloud/utils.pm

```diff
@@ -885,4 +887,92 @@ sub upload_asset_on_remote {
     $instance->ssh_assert_script_run($mv_cmd);
 }
 
+
+=head2 zypper_remote_call
+
+    zypper_remote_call($command [, exitcode => $exitcode] [, timeout => $timeout];
+
+Function wrapping zypper or transactional-update command for remote execution via ssh; not for tunneling.
+Implements lib/utils::zypper_call, dedicated to publiccloud instances, but the input command is totally up to the user, 
```

Comment:
This is against the goal of the [ticket poo#188991](https://progress.opensuse.org/issues/188991) and exhibits feature creep.

The overall goal was to implement a `$instance->zypper_call` function analog to the already existing `zypper_call`.

Reply:
updated